src/lib/viz/style/helpers/size-categories-style.ts

```diff
@@ -112,12 +115,16 @@ function calculateWithCategories(
   let obj;
 
   if (geometryType === 'Point') {
-    obj = {
-      getRadius: getSizeValue,
-      pointRadiusMinPixels: minSize,
-      pointRadiusMaxPixels: maxSize,
-      radiusUnits: 'pixels'
-    };
+    if (!options.property || options.property === 'size') {
+      obj = {
+        getRadius: getSizeValue,
+        pointRadiusMinPixels: minSize,
+        pointRadiusMaxPixels: maxSize,
+        radiusUnits: 'pixels'
+      };
+    } else if (options.property === 'strokeWidth') {
+      obj = { getLineWidth: getSizeValue };
```

Comment:
Should we set the line widht Units param to pixels too?

Reply:
Done